src/selectedtests/app/controllers/project_test_mappings_controller.py

```diff
@@ -19,7 +19,7 @@ def add_project_test_mappings_endpoints(api: Api, mongo: MongoWrapper, evg_api:
     :param mongo: Mongo Wrapper instance
     :param evg_api: An instance of the evg_api client
     """
-    ns = api.namespace("projects", description="Project Test Mappings")
+    ns = api.namespace("projects", description="Project Mappings")
```

Comment:
This namespace contains both test and task mappings so I made the description more general:
![Screen Shot 2019-11-21 at 3 51 39 PM](https://user-images.githubusercontent.com/7087489/69375715-f556c480-0c76-11ea-9f95-8cf4c8b596c4.png)